pages/common/lmms.md

```diff
@@ -0,0 +1,36 @@
+# lmms
+
+> Render a `.mmp` or `.mmpz` project file, dump a `.mmpz` as XML, or start the GUI.
```

Comment:
I think this description is a little low level.  A new user likely doesn't know what a `.mmp` or `.mmpz` file is (I didn't :smile:)
```suggestion
> Free, open source, multiplatform digital audio workstation.
```

Reply:
ah, i was going under the assumption that LMMS users would already know the file formats, but you have a point :D

Reply:
Of course, but _probably_ most users will see this page by running `tldr lmms` _because_ they have no idea what it is.

Reply:
That's the way we think when we write page titles, usually.